index.ts

```diff
@@ -0,0 +1,44 @@
+import { RGBot } from "rg-bot";
+
+/**
+ * This strategy is the simplest example of how to get started with the rg-bot package.
+ * The Bot will run around and gather Poppies until it has 100 in its inventory.
+ */
+export function configureBot(bot: RGBot) {
```

Comment:
I should have put this same comment in the other PRs... The actual signature of configureBot is below.  We pass in the matchInfo emitter so they can subscribe to events like `player_joined, match_ended, match_started, player_left`

```javascript
/**
 * @param {RGBot} bot
 * @param {EventEmitter} matchInfoEmitter
 */
function configureBot(bot, matchInfoEmitter) {

```

Reply:
Added this to the samples for now. Since the RGBot instance we're passing into configureBot already has a reference to the matchInfoEmitter, I think it'd be better in the future to add a method to RGBot for subscribing to matchInfo events instead of passing it as a separate param

Reply:
If we're going to go that route, we shouldn't put this into the templates only to remove it later.

Reply:
Can you show a sample of what you envision the new code would look like ?

Reply:
reverted

Reply:
Since we own the matchEmitter we could add a method per event rather than the more generic approach we took for mineflayer.on()

ex. `rgBot.onMatchStart(() => { // do something })`
same for `onPlayerJoin`, `onPlayerLeave` etc.

Reply:
as long as this also allows removing the listeners dynamically